tracing-api/build.gradle

```diff
@@ -1,9 +1,6 @@
 apply from: "$rootDir/gradle/publish-jar.gradle"
 
 dependencies {
-    compile "com.fasterxml.jackson.core:jackson-databind"
-    implementation 'com.fasterxml.jackson.core:jackson-annotations'
```

Comment:
What would happen if we made these `compileOnly`, so that they don't appear in the `compile` or `runtime` sections of the POM? I think consumers who had jackson on the classpath would still be able to work?

Reply:
It would avoid breaking consumers which serialize Span to json, but I'm curious if that's something anyone does, and if so, is it something that they should be able to do?

I think the more likely break is excavator failures when tracing-api no longer provides jackson-databind, but it's only likely to affect a few libraries with relatively few dependencies. The exact dependencies plugin protects consumers from this, and it's easy to fix, so it's not something I would optimize on.